ntoskrnl/kdbg/kdb_cli.c

```diff
@@ -2662,15 +2662,16 @@ KdbpPrint(
         {
             while ((p2 = strrchr(p, '\x1b'))) /* Look for escape character */
             {
+                size_t len = strlen(p2);
                 if (p2[1] == '[')
                 {
                     j = 2;
                     while (!isalpha(p2[j++]));
-                    strcpy(p2, p2 + j);
+                    memmove(p2, p2 + j, len + 1 - j);
```

Comment:
Maybe something like:
`memmove(p2, p2 + j, Length - (p2 - Buffer));`
may work

Reply:
I don't see how this is any better.